ebpf-for-windows-undock.sln

```diff
@@ -0,0 +1,3235 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.4.33205.214
```

Comment:
nit: can be build ebpf using this `sln` file in GitHub runners? If yes, should we add a build step in GitHub CICD using this `sln` file?

That will help validate that any changes done to this file in a GitHub PR are correct.

Reply:
Done.